src/linear-algebra.lisp

```diff
@@ -259,5 +259,5 @@ The function will just return NIL, and modify the contents of RESULT."
                    :name (concatenate 'string (quil:gate-name gate) "*")
                    :dimension (quil:gate-dimension gate)
                    :matrix-function #'(lambda (&rest parameters)
-                                        (magicl:conjugate-entrywise
-                                         (apply #'quil:gate-matrix gate parameters))))))
+                                        (magicl:map! #'conjugate
+                                                     (apply #'quil:gate-matrix gate parameters))))))
```

Comment:
ditto here regarding mutation, although I think `gate-matrix` on a `parameterized-gate` will return a fresh matrix, so maybe this is ok.

Reply:
Yeah this should return a fresh matrix so `map!` should be the faster option